7-high-level-structural-patterns.rst

```diff
@@ -1172,6 +1172,23 @@ Examples
 Images
 ******
 
+#.	:html:`<img>` Each image has a unique :html:`id` attribute.
```

Comment:
Is the `<img>` tag supposed to lead that sentence? Can we rephrase this so that it's a complete sentence?